File: APL1.2/TPStructures/date.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>

int main(void){
	int jour,mois,annee;
	time_t now;

	time(&now);
	struct tm *local = localtime(&now);
	jour = local->tm_mday;          
	mois = local->tm_mon + 1;     
	annee = local->tm_year + 1900;



	printf("Date : %d / %d / %d\n",jour,mois,annee);
	return 0;
}

File: APL2.1/TP2/Date.java
public class Date {
	int jour;
	int mois;
	int annee;


	public Date() {
		this.jour = 30;
		this.mois = 01;
		this.annee = 2020;
	}

	



	public String toString(){
		return(this.jour+"-"+this.mois+"-"+this.annee);

	}
	public static void main(String[] args) {
		Date c = new Date();
		System.out.println("Date du jour : " + c);
	
	

}
}